validator/client/validator.go

```diff
@@ -141,8 +143,7 @@ func (v *validator) WaitForKeymanagerInitialization(ctx context.Context) error {
 }
 
 // subscribe to channel for when the wallet is initialized
-func waitForWebWalletInitialization(ctx context.Context, walletInitializedEvent *event.Feed) (keymanager.IKeymanager, error) {
-	walletChan := make(chan *wallet.Wallet)
+func waitForWebWalletInitialization(ctx context.Context, walletInitializedEvent *event.Feed, walletChan chan *wallet.Wallet) (keymanager.IKeymanager, error) {
```

Comment:
See: https://gobyexample.com/channel-directions, also this line is too long, so we should instead format it as
```
func foo(
  a type,
  b type,
) (returnType, error)
```

Reply:
change made